org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -1289,7 +1290,7 @@ class FileEvent {
 /**
  * Value-object describing what options formatting should use.
  */
-class FormattingOptions extends LinkedHashMap<String, Either<String, Either<Integer, Boolean>>> {
+class FormattingOptions extends LinkedHashMap<String, Either3<String, Integer, Boolean>> {
```

Comment:
I thought `Number` should be used instead of `Integer`? I think the spec allows all kind of numbers. But it is a separate issue.

Reply:
I changed that to `Integer` a while ago because of [a review comment](https://github.com/eclipse/lsp4j/pull/110#discussion_r129280523) by @svenefftinge, but I agree that Number should be correct here.